src/renderer/PresetsBar.tsx

```diff
@@ -44,6 +45,7 @@ const PresetListItem = ({
   };
 
   const handleDeleteClicked = () => {
+    deletePreset(value);
```

Comment:
could we move this into `handleDeletePreset` in the outer component and add an `await` + `try/catch` around `deletePreset`?

Reply:
yeah